x-pack/plugins/oss_telemetry/server/test_utils/index.ts

```diff
@@ -4,9 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { APICaller, CoreSetup } from 'kibana/server';
+import { APICaller } from 'kibana/server';
 
 import { of } from 'rxjs';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
+import { elasticsearchServiceMock } from 'src/core/server/mocks';
```

Comment:
Obviously not the case, but I though we did allow imports from `src/core/server/mocks` @restrry?

Reply:
We do https://github.com/elastic/kibana/blob/35302ed2731404bb2697029314d7bb45b0c49a6b/.eslintrc.js#L231

Reply:
Hum strange, I got an eslint error when removing the ignore

Reply:
There are problems with a path resolution for absolute paths. Probably that it is https://github.com/elastic/kibana/issues/36096

Reply:
Fixed by switching to an absolute path.